src/main/java/com/eerussianguy/firmalife/jei/DryingRecipeWrapper.java

```diff
@@ -22,7 +22,11 @@ public void drawInfo(Minecraft minecraft, int recipeWidth, int recipeHeight, int
     {
         float x = 60f;
         float y = 4f;
-        String text =  DryingRecipe.getDuration(recipe) / ICalendar.TICKS_IN_DAY + " " + I18n.format("tooltip.firmalife.days");
+        String text = (DryingRecipe.getDuration(recipe) / ICalendar.TICKS_IN_HOUR < 24) ?
```

Comment:
I might make this <= 48. For times less than 2 days, it might be that it makes more sense to show a more instructive hour count.